### PR TITLE
fluentd v1.11.4

### PR DESCRIPTION
**Pull request by repeatedly**
Signed-off-by: Masahiro Nakagawa <repeatedly@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,42 +12,42 @@
...
+	v1.11/arm64/debian-stackdriver:v1.11.4-debian-stackdriver-arm64-1.0,v1.11-debian-stackdriver-arm64-1 \
+	v1.11/arm64/debian-s3:v1.11.4-debian-s3-arm64-1.0,v1.11-debian-s3-arm64-1 \
+	v1.11/arm64/debian-syslog:v1.11.4-debian-syslog-arm64-1.0,v1.11-debian-syslog-arm64-1 \
+	v1.11/arm64/debian-forward:v1.11.4-debian-forward-arm64-1.0,v1.11-debian-forward-arm64-1 \
+	v1.11/arm64/debian-gcs:v1.11.4-debian-gcs-arm64-1.0,v1.11-debian-gcs-arm64-1 \
+	v1.11/arm64/debian-graylog:v1.11.4-debian-graylog-arm64-1.0,v1.11-debian-graylog-arm64-1 \
+	v1.11/arm64/debian-papertrail:v1.11.4-debian-papertrail-arm64-1.0,v1.11-debian-papertrail-arm64-1 \
+	v1.11/arm64/debian-logzio:v1.11.4-debian-logzio-arm64-1.0,v1.11-debian-logzio-arm64-1 \
+	v1.11/arm64/debian-kafka:v1.11.4-debian-kafka-arm64-1.0,v1.11-debian-kafka-arm64-2 \
+	v1.11/arm64/debian-kinesis:v1.11.4-debian-kinesis-arm64-2.0,v1.11-debian-kinesis-arm64-2
```
Comment (cosmo0920) [inline Makefile:50]: Using `-2.0` suffix is reasonable?
I'd bumped fluent-plugin-kinesis plugin version to v3.2.3 in the past few weeks.

File: docker-image/v1.11/arm64/debian-azureblob/Dockerfile [hunk 2]
```diff
@@ -8,7 +8,7 @@ ENV QEMU_DOWNLOAD_SHA256 a1ef52971537e11915565233f48aa179839f676008d7911c05b3ae9
 RUN apk add curl --no-cache
 RUN curl -sL -o qemu-3.0.0+resin-aarch64.tar.gz https://github.com/balena-io/qemu/releases/download/v3.0.0%2Bresin/qemu-3.0.0+resin-aarch64.tar.gz && echo "$QEMU_DOWNLOAD_SHA256 *qemu-3.0.0+resin-aarch64.tar.gz" | sha256sum -c - | tar zxvf qemu-3.0.0+resin-aarch64.tar.gz -C . && mv qemu-3.0.0+resin-aarch64/qemu-aarch64-static .
 
-FROM fluent/fluentd:v1.11.3-debian-arm64-1.1
+FROM fluent/fluentd:v1.11.4-debian-arm64-1.1
```
Comment (cosmo0920) [inline docker-image/v1.11/arm64/debian-azureblob/Dockerfile:11]: Off topic: I'd bumped suffix version to `1.1` for applying debian slim image for armv7 and arm64:
* https://github.com/fluent/fluentd-docker-image/commit/abc3dcf0d12fc6087c1fe5e01eb1a39ca24799a6#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52
* https://github.com/fluent/fluentd-docker-image/commit/555f1eb2e8ea12dffd32a24853b6911da6385403#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52

We should keep as-is for now....

**Review by cosmo0920 — COMMENTED** [review]
I'd commented a few things.
Could you check them?

File: Makefile (hunk 1 above)
Comment (repeatedly) [reply Makefile:50]: What is the proper version? `-1.0`?
Could you change the version in this branch?

File: docker-image/v1.11/arm64/debian-azureblob/Dockerfile (hunk 2 above)
Comment (repeatedly) [reply docker-image/v1.11/arm64/debian-azureblob/Dockerfile:11]: I will use `1.0` for v1.11.4 docker image.

File: Makefile (hunk 1 above)
Comment (cosmo0920) [reply Makefile:50]: Sure. Will do.

**Comment by cosmo0920** [comment]
I'd updated to use 1.0 suffix for all newer Fluentd images.

**Pull request merged**